src/common/update.rs

```diff
@@ -39,7 +38,7 @@ pub struct UpdateParams {
     #[serde(default)]
     pub ordering: WriteOrdering,
     #[serde(default)]
-    pub timeout: Option<NonZeroU64>,
+    pub timeout: Option<Duration>,
```

Comment:
We might need something like this (untested) from [`serde_with`](https://docs.rs/serde_with/):

```suggestion
    #[serde_as(as = "DurationSeconds<String>")]
    pub timeout: Option<Duration>,
```